selfdrive/car/hyundai/carcontroller.py

```diff
@@ -0,0 +1,174 @@
+from common.numpy_fast import clip, interp
+from selfdrive.boardd.boardd import can_list_to_can_capnp
+from selfdrive.car.hyundai.hyundaican import make_can_msg, create_lkas11, create_lkas12b
+from selfdrive.car.hyundai.values import ECU, STATIC_MSGS, CAR
+from selfdrive.can.packer import CANPacker
+
+
+# Steer torque limits
+STEER_MAX = 200   # Actual limit is about 1023, but not tested, and not needed
```

Comment:
so, I assume 1023 is absurdly high? would be nice to add the 200 limit so the safety_hyundai.h in panda

Reply:
It might be safe to push it to 250, but it is VERY strong, any more would be unsafe.
1023 is the maximum due to the size of the integer, but it does not accept messages over 767

Reply:
Then I definitely recommend to add the 200 limit panda safety.